bundles/org.openhab.binding.elroconnects/src/main/java/org/openhab/binding/elroconnects/internal/handler/ElroConnectsBridgeHandler.java

```diff
@@ -528,7 +544,21 @@ private void processAlarmTriggerMessage(ElroConnectsMessage message) {
         if (handler != null) {
             handler.triggerAlarm();
         }
+        // Also trigger an alarm on the bridge, so the alarm also comes through when no thing for the device is
+        // configured
+        triggerChannel(ALARM, Integer.toString(deviceId));
         logger.debug("Device ID {} alarm", deviceId);
+
+        if (answerContent.length() < 22) {
+            logger.debug("Could not get device status from alarm message for device {}", deviceId);
+            return;
+        }
+        String deviceStatus = answerContent.substring(14, 22);
```

Comment:
I don't know how this string looks, but interpretation seems to carry some assumptions. Can it be sanity checked somehow?

Reply:
The content of the device status string is content specific. It is mapped and sanity checked in the ElroConnectsDevice class and its concrete subclasses. I can only pass on the info here if I don't want to bring all the device specific logic in this class.

Reply:
Hmm, you mean `deviceStatus` is passed to the concrete device classes below? I'm referring to the original string, `answerContent`. What are in the first skipped characters? How do you know how many to skip? Is it ASCII content only?

Reply:
I know by looking at the wiresharks of the responses and other code available on the internet. So far, I have only been able to observe ASCII strings. Characters 6 to 10 are relevant and interpreted before (deviceId). 10 to 14 contains equipment type, but that is already known in the binding, so not relevant. Not everything in the message format is always clear, but this interpretation has worked so far.

Reply:
Okay, seems like a crucial part of the protocol which you have fully under control, so let's leave it like it is. :)